docs/getting-started/quickstart-programming.md

```diff
@@ -3,29 +3,19 @@ sidebar_position: 2
 title: Programming Guide
 ---
 
-Let's learn how to build applications with [DBOS Transact](https://github.com/dbos-inc/dbos-ts).
-In this tutorial, we'll modify the example application from our [quickstart](./quickstart.md) to reliably send greetings to your friends.
-We'll show you how construct a reliable workflow that updates your database and calls third-party APIs.
+Before starting this tutorial, we recommend finishing the [quickstart](./quickstart.md). We'll modify the quickstart example to illustrate the core building blocks of [DBOS Transact](https://github.com/dbos-inc/dbos-ts). In the end, we'll create an app that reliably records events across two different systems: the Postgres database and a third-party API. The app will write to both systems consistently, even if we crash it between writes.
```

Comment:
Maybe also mention "We'll walk you through how to set up the app and its database to run it locally"

Reply:
Rewritten; take a look